src/pages/run-a-node.js

```diff
@@ -0,0 +1,1011 @@
+  gap: 1rem;
+  margin-top: 2rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    flex-direction: column;
+  }
+`
+
+const DappNodeButtonLink = styled(ResponsiveButtonLink)`
+  background-color: #30bcb2;
+  span {
+    color: ${({ theme }) => theme.colors.white};
+  }
+  &:hover {
+    background-color: #3ec3c6;
+    box-shadow: 4px 4px 0px 0px #3ec3c677;
+  }
+`
+
+const AvadoButtonLink = styled(ResponsiveButtonLink)`
+  background-color: #4a9b40;
+  span {
+    color: ${({ theme }) => theme.colors.white};
+  }
+  &:hover {
+    background-color: #5baa4a;
+    box-shadow: 4px 4px 0px 0px #5baa4a77;
+  }
+`
+
+const StyledEmoji = styled(Emoji)`
+  margin-right: 1rem;
+`
+
+const ScrollLink = styled(NakedButton)`
+  text-align: start;
+  color: ${({ theme }) => theme.colors.primary};
+  &.active {
+    color: ${({ theme }) => theme.colors.primary};
+  }
+`
+
+const ScrollButtonSecondary = styled.button`
+  text-decoration: none;
+  display: inline-block;
+  padding: 0.5rem 0.75rem;
+  font-size: 1rem;
+  border-radius: 0.25em;
+  text-align: center;
+
+  color: ${({ theme }) => theme.colors.text};
+  border: 1px solid ${({ theme }) => theme.colors.text};
+  background-color: transparent;
+
+  &:hover {
+    color: ${({ theme }) => theme.colors.primary};
+    border: 1px solid ${({ theme }) => theme.colors.primary};
+    box-shadow: ${({ theme }) => theme.colors.cardBoxShadow};
+  }
+  &:active {
+    background-color: ${({ theme }) =>
+      theme.colors.secondaryButtonBackgroundActive};
+  }
+`
+
+const DiscordIcon = styled(Icon)`
+  fill: ${({ theme }) => theme.colors.buttonColor};
+`
+
+const StakingCalloutContainer = styled(SplitContent)`
+  background: linear-gradient(
+    262.78deg,
+    rgba(152, 186, 249, 0.25) 0%,
+    rgba(207, 177, 251, 0.25) 53.12%,
+    rgba(151, 252, 246, 0.25) 100%
+  );
+  width: 100%;
+  padding: 2rem;
+  gap: 5rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+    gap: 3rem;
+  }
+`
+
+const Leslie = styled(GatsbyImage)`
+  transform: scaleX(-1) scale(1.15) translateX(2rem);
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    transform: scaleX(-1) translateY(-3rem);
+  }
+`
+
+const StyledFeedbackCard = styled(FeedbackCard)`
+  width: 100%;
+  max-width: 700px;
+  margin: 0 2rem;
+`
+
+const RunANodePage = ({ data }) => {
+  const intl = useIntl()
+  const heroContent = {
+    title: <Translation id="page-run-a-node-title" />,
+    header: <Translation id="page-run-a-node-hero-header" />,
+    subtitle: <Translation id="page-run-a-node-hero-subtitle" />,
+    image: getImage(data.ethereumInside),
+    alt: <Translation id="page-run-a-node-hero-alt" />,
+    buttons: [
+      {
+        content: <Translation id="page-run-a-node-button-why-run-a-node" />,
+        pathId: "why-run-a-node",
+      },
+      {
+        content: <Translation id="page-run-a-node-button-get-started" />,
+        pathId: "getting-started",
```

Comment:
I don't think we should have two primary action buttons here. [More info on best practices.](https://balsamiq.com/learn/articles/button-design-best-practices/#2-visually-differentiate-primary-and-secondary-actions)

```suggestion
        pathId: "getting-started",
        isSecondary: "isSecondary",
```